This patch was developed in-house, offered upstream, but rejected because
of a difference of opinion about typical settings for font and screen sizes.

--- xscreensaver-5.15/hacks/glx/photopile.c.1	2011-09-08 23:52:32.000000000 -0700
+++ xscreensaver-5.15/hacks/glx/photopile.c	2014-05-18 19:39:24.308851725 -0700
@@ -626,7 +626,7 @@
       const char *title = frame->title ? frame->title : "(untitled)";
 
       /* #### Highly approximate, but doing real clipping is harder... */
-      int max = 35;
+      int max = 61;
       if (strlen(title) > max)
         title += strlen(title) - max;
 
